src/test/scala/uk/gov/hmrc/hmrcfrontend/views/helpers/hmrcLayoutSpec.scala

```diff
@@ -321,6 +322,18 @@ class hmrcLayoutSpec extends AnyWordSpecLike with Matchers with JsoupHelpers wit
       customMainContent.childrenSize() shouldBe 0
     }
 
+    "not use TwoThirdsMainContent layout if passed main content layout is None" in {
+      val hmrcLayout = app.injector.instanceOf[HmrcLayout]
+      val messages   = getMessages()
+      val content    = contentAsString(
+        hmrcLayout(mainContentLayout = None)(Html("Some main content"))(fakeRequest, messages)
+      )
+      val document   = Jsoup.parse(content)
+
+      val mainContent = document.getElementById("main-content")
+      Try(s"Found unexpected content wrapper ${mainContent.child(0).outerHtml()}").toOption shouldBe None
```

Comment:
`Try`ing to provide meaningful context if it fails, but not throw if all is well